### DISCUSSION TITLE
How to add new record with edit row mode?

### DISCUSSION THREAD

**Discussion by anilrepala505** [primefaces/community#1292] [PrimeVue] [answered]
With edit row mode, we can edit a row. Can we add a new row in same way? Like creating an empty row in the table with options to save and cancel.

**Comment by tugcekucukoglu** [comment, marked as answer]
This example could be helpful https://primevue.org/datatable/#dtproducts

**Reply by ddrm86** [reply]
Is the link broken? It takes me to the beginning of the DataTable documentation and I cannot find an example to add a new record.
